lib/amazonka-core/test/Test/Amazonka/Sign/V4/Base.hs

```diff
@@ -104,8 +104,7 @@ mkV4Paths genAbbrev genPath = do
           & requestService .~ svc
           & requestBody .~ Hashed aBody
 
-  (meta, _) <- base (Tag (sha256Base16 aBody)) req <$> arbitrary <*> arbitrary <*> arbitrary
-  pure (aPath, metaPath meta, metaCanonicalPath meta)
+  pure (aPath, escapedPath req, canonicalPath req)
```

Comment:
The tradeoff here is the test ends up testing at a lower level, testing escapedPath and canonicalPath separately. Whether those two are actually used to compute the http request path and the sigv4 canonical path isn't covered by the tests.

Reply:
I'm comfortable with that. The facet you're talking about should properly be tested by computing full signatures somewhere, but I'm happy to merge without requiring that sort of test.

Reply:
Yeah that makes sense.